### PR TITLE
create header component

### PR DESCRIPTION
**Pull request by julienmachon**
fixes BlueBrain/nexus#241

screenshot from 2018-11-02 16-42-22

### CODE REVIEW

File: src/server/html.ts [hunk 1]
```diff
@@ -14,7 +15,9 @@ const html = ({ body, helmet }: { body: string, helmet: HelmetData }): string =>
       ${helmet.title.toString()}
       ${helmet.meta.toString()}
       ${helmet.link.toString()}
+      <link rel="shortcut icon" type="image/png" href="${icon}"/>
       <link rel="stylesheet" href="${base}public/bundle.css" />
+      <meta name="viewport" content="width=device-width, initial-scale=1">
```
Comment (mdupont) [inline src/server/html.ts]: Nitpicking, but grouping <link> and <meta> tags together makes it better organized (with the exception of the charset <meta> tag which should come before the <title> if we need to have it.

So `${helmet.meta.toString()}` could be moved below the latest <link> and above the viewport <meta>

File: src/shared/components/Header/Header.less [hunk 2]
```diff
@@ -1,7 +1,54 @@
 .Header {
-  background-color: #ff6666;
+  display: flex;
+  justify-content: space-between;
+  height: 60px;
+  background-color: #f0f0f0;
```
Comment (mdupont) [inline src/shared/components/Header/Header.less]: variables should be preferred to plain values

**Comment by julienmachon** [comment]
Agreed! I just need to figure out how to make them work with my webpack setup 😬

File: src/shared/components/Header/Header.less (hunk 2 above)
Comment (julienmachon) [reply src/shared/components/Header/Header.less]: Agreed! I just need to figure out how to make them work with my webpack setup 😬

File: src/server/html.ts (hunk 1 above)
Comment (julienmachon) [reply src/server/html.ts]: Fair enough, I like things to be tidy as well

**Pull request merged**